server/src/routes/items.routes.ts

```diff
@@ -48,6 +48,22 @@ router.get('/:categoryId/', authenticate, async (req: Request, res: Response) =>
     res.status(statusCode).send(response);
 });
 
+router.get('/delete/:itemId/', authenticate, async (req: Request, res: Response) => {
```

Comment:
Add validations for:
- itemId exists in DB with the given userId. You can reuse it in both DELETE and PUT method if you'll extract the field like this:
```
body('_id', 'Invalid itemId')
  .custom(async (_id: string | undefined, {req}: {req: Request}) => {
    const itemId: string = _id || req.body.itemId;
    // Complete validation
  }
```